examples/pump-app/nrfconnect/CMakeLists.txt

```diff
@@ -16,13 +16,30 @@
 cmake_minimum_required(VERSION 3.13.1)
 
 get_filename_component(CHIP_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/third_party/connectedhomeip REALPATH)
+get_filename_component(NLIO_ROOT ${CHIP_ROOT}/third_party/nlio/repo/include REALPATH)
 get_filename_component(NRFCONNECT_COMMON ${CHIP_ROOT}/examples/platform/nrfconnect REALPATH)
-get_filename_component(PUMP_COMMON ${CHIP_ROOT}/examples/pump-app/pump-common REALPATH)
+get_filename_component(PUMP_COMMON ${CHIP_ROOT}/examples/pump-controller-app/pump-controller-common REALPATH)
 
 include(${CHIP_ROOT}/config/nrfconnect/app/check-nrfconnect-version.cmake)
 
 # Load NCS/Zephyr build system
 set(CONF_FILE ${CHIP_ROOT}/config/nrfconnect/app/sample-defaults.conf prj.conf)
+
+if (EXISTS boards/${BOARD}.conf)
+  list(APPEND CONF_FILE boards/${BOARD}.conf)
+endif()
+
+# TODO: temporary fix to remove after solving static addressing problem on nrf5340
+if(${BOARD} STREQUAL "nrf5340dk_nrf5340_cpuapp")
+    list(INSERT OVERLAY_CONFIG 0 ${CHIP_ROOT}/config/nrfconnect/app/overlay-bt_private_addresses.conf)
+endif()
+
+option(BUILD_WITH_DFU "Build target with Device Firmware Upgrade support" OFF)
```

Comment:
I think currently this option will not work - there is no `pm_static.yml`.

Reply:
Fixed in latest commit